### PR TITLE
CypressE2E: Fixed Test Cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
Fixed Sidebar and Display mode test cases

#### Screenshots
image

#### Release Note
```
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/account_settings/display/message_display_mode_spec.js [hunk 1]
```diff
@@ -64,11 +64,11 @@ function verifyLineBreaksRemainIntact(display) {
...
 
         // * Verify HTML includes newline and the edit
-        cy.get(postMessageTextId).should('have.html', `<p>${firstLine}</p>\n<p>${secondLine},edited</p>`);
+        cy.get(postMessageTextId).should('contain.html', `<p>${firstLine}</p>\n<p>${secondLine},edited <span`);
 
         // * Post should have (edited)
         cy.get(`#postEdited_${postId}`).
             should('be.visible').
-            should('contain', '(edited)');
+            should('contain', 'Edited');
```
Comment (saturninoabril) [inline e2e/cypress/integration/account_settings/display/message_display_mode_spec.js:72]: Oh no, I got this covered at https://github.com/mattermost/mattermost-webapp/pull/9006 as well.

**Review by saturninoabril — APPROVED** [review]
Thanks @furqanmlk!

File: e2e/cypress/integration/account_settings/display/message_display_mode_spec.js (hunk 1 above)
Comment (furqanmlk) [reply e2e/cypress/integration/account_settings/display/message_display_mode_spec.js:72]: Oh sorry, I should have informed you, before I fix them :(

**Review by josephbaylon — APPROVED** [review]
LGTM

File: e2e/cypress/integration/account_settings/display/message_display_mode_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/account_settings/display/message_display_mode_spec.js:72]: No worries. I'm good as is.

**Pull request merged**